pyspecProcScripts/integral_w_error.py

```diff
@@ -147,9 +149,12 @@ def active_propagation(
         )
     s_forerror = select_pathway(s, signal_path)
     N = psp.ndshape(s_forerror)[direct]
+    s_forerror -= s_forerror.C.mean_all_but([indirect,direct]).mean(direct)
     s_forerror.run(np.real).run(lambda x: abs(x) ** 2).mean_all_but(
-        [direct, indirect]
-    ).mean(direct)
+        [direct, indirect]).mean(direct)
     s_forerror *= df ** 2
     s_forerror *= N
-    return s_forerror.run(psp.sqrt)
+    s_full = select_pathway(full_s,signal_path)
+    retval = s_full.integrate(direct).set_error(psp.sqrt(s_forerror.data))
+    return retval
```

Comment:
what is the point of this change? when we talk, you'll have  to explain it to me.